### PR TITLE
bgpgrep, peerindex: add German translation

### PR DESCRIPTION
**Pull request by marek22k**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.de/common/bgpgrep.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# bgpgrep
+
+> Filtere und Gebe BGP-Daten in einem MRT-Dump aus.
+> Kann mit gzip, bzip2 und xz komprimierte Dateien lesen.
+> Weitere Informationen: <https://gitea.it/1414codeforge/ubgpsuite>.
+
+- Gebe alle Routen aus:
```
Comment (pixelcmtd) [inline pages.de/common/bgpgrep.md:7]:
```suggestion
- Gib alle Routen aus:
```

File: pages.de/common/bgpgrep.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# bgpgrep
+
+> Filtere und Gebe BGP-Daten in einem MRT-Dump aus.
```
Comment (pixelcmtd) [inline pages.de/common/bgpgrep.md:3]:
```suggestion
> Filtere BGP-Daten in einem MRT-Dump und gebe sie aus.
```

File: pages.de/common/bgpgrep.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+> Kann mit gzip, bzip2 und xz komprimierte Dateien lesen.
+> Weitere Informationen: <https://gitea.it/1414codeforge/ubgpsuite>.
+
+- Gebe alle Routen aus:
+
+`bgpgrep {{master6.mrt}}`
+
+- Gebe alle von einem bestimmten Peer empfangenen Routen aus, bestimmt durch die AS-Nummer des Peers:
```
Comment (pixelcmtd) [inline pages.de/common/bgpgrep.md:11]:
```suggestion
- Gib alle von einem bestimmten Peer empfangenen Routen aus, bestimmt durch die AS-Nummer des Peers:
```

File: pages.de/common/bgpgrep.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+`bgpgrep {{master4.mrt}} -peer {{64498}}`
+
+- Gebe alle von einem bestimmten Peer empfangenen Routen aus, bestimmt durch die IP-Adresse des Peers:
```
Comment (pixelcmtd) [inline pages.de/common/bgpgrep.md:15]:
```suggestion
- Gib alle von einem bestimmten Peer empfangenen Routen aus, bestimmt durch die IP-Adresse des Peers:
```

File: pages.de/common/bgpgrep.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+`bgpgrep {{master4.mrt.bz2}} -peer {{2001:db8:dead:cafe:acd::19e}}`
+
+- Gebe alle Routen aus, die bestimmte ASNs in ihrem AS-Pfad haben:
```
Comment (pixelcmtd) [inline pages.de/common/bgpgrep.md:19]:
```suggestion
- Gib alle Routen aus, die bestimmte ASNs in ihrem AS-Pfad haben:
```

File: pages.de/common/bgpgrep.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+`bgpgrep {{master6.mrt.bz2}} -aspath '{{64498 64510}}'`
+
+- Gebe Routen aus, die zu einer bestimmten Adresse führen:
```
Comment (pixelcmtd) [inline pages.de/common/bgpgrep.md:23]:
```suggestion
- Gib Routen aus, die zu einer bestimmten Adresse führen:
```

File: pages.de/common/bgpgrep.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+`bgpgrep {{master6.mrt.bz2}} -supernet '{{2001:db8:dead:cafe:aef::5}}'`
+
+- Gebe alle Routen aus, die Communities von einem bestimmten AS haben:
```
Comment (pixelcmtd) [inline pages.de/common/bgpgrep.md:27]:
```suggestion
- Gib alle Routen aus, die Communities von einem bestimmten AS haben:
```

File: pages.de/common/peerindex.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
+# peerindex
+
+> Liest MRT TABLE_DUMPV2 Peer Index Table aus.
+> Kann mit gzip, bzip2 und xz komprimierte Dateien lesen.
+> Weitere Informationen: <https://gitea.it/1414codeforge/ubgpsuite>.
+
+- Gebe alle Peers aus:
```
Comment (pixelcmtd) [inline pages.de/common/peerindex.md:7]:
```suggestion
- Gib alle Peers aus:
```

File: pages.de/common/peerindex.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+
+`peerindex {{master6.mrt}}`
+
+- Anzeige aller Peers, die Routing-Informationen bereitgestellt haben:
```
Comment (pixelcmtd) [inline pages.de/common/peerindex.md:11]:
```suggestion
- Zeige alle Peers an, die Routing-Informationen bereitgestellt haben:
```

**Review by pixelcmtd — COMMENTED** [review]
Despite both being grammatically correct (afaik), we have actually never used „Gebe“, while there are many pages with „Gib“

**Comment by marek22k** [comment]
Thanks for the suggestion.

**Review by pixelcmtd — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**